src/testUtils/TestRig.react.js

```diff
@@ -54,6 +54,11 @@ export default class TestRig {
     var element = this.domNode.find(selector)
     this.fillInElement(element, value)
   }
+  toggleCheckbox(selector) {
+    var element = this.domNode.find(selector) // Selector should be specific to the checkbox
```

Comment:
@maneeshnycda How would you feel about changing this to:
```jsx
var element = $(this.domNode.find(selector))
```
?
That way the next line doesn't have to wrap `element` in `$()` twice--it's already done.